### PR TITLE
Add Some Unittest

### PR DESCRIPTION
**Pull request by thecodejit**
- Add test for auth_controller

### CODE REVIEW

**Comment by canihavesomecoffee** [comment]
I'll check after the weekend.

**Comment by thecodejit** [comment]
Anytime!! 😄😃

**Comment by canihavesomecoffee** [comment]
@pep8speaks suggest diff

File: tests/tests_auth/test_controllers.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+class TestLogIn(BaseTestCase):
+
+    def test_if_login_form_renders(self):
+        response = self.app.test_client().get(url_for('auth.login'))
+        self.assertEqual(response.status_code, 200)
+        self.assert_template_used('auth/login.html')
+
+    def test_login(self):
+
```
Comment (canihavesomecoffee) [inline tests/tests_auth/test_controllers.py:14]: This is empty?

File: tests/tests_auth/test_controllers.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+class TestLogOut(BaseTestCase):
```
Comment (canihavesomecoffee) [inline tests/tests_auth/test_controllers.py:17]: Did you actually run these tests? There is no way that this test currently can pass.

File: tests/main.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            'SQLALCHEMY_POOL_SIZE': 1,
+            'SECRET_KEY': secret_key,
+            'CSRF_SESSION_KEY': secret_csrf
+            }
+
+
+class BaseTestCase(TestCase):
+    @mock.patch('config_parser.parse_config', side_effect=load_config)
+    def create_app(self, mock_config):
+        role = namedtuple('role', 'name')
```
Comment (canihavesomecoffee) [inline tests/main.py:46]: Why the use of namedtuple?

**Review by canihavesomecoffee — CHANGES_REQUESTED** [review]

File: tests/tests_auth/test_controllers.py (hunk 2 above)
Comment (thecodejit) [reply tests/tests_auth/test_controllers.py:17]: Working on it and getting it fixed.

**Pull request closed without merging**